docs/tempo/website/operations/monitoring.md

```diff
@@ -5,12 +5,45 @@ weight: 4
 
 # Monitoring Tempo
 
-The Tempo repository has a [mixin](https://github.com/grafana/tempo/tree/main/operations/tempo-mixin) that includes a
-set of dashboards, rules and alerts. Together, these can be used to monitor Tempo in production.
+Tempo is instrumented to expose metrics, logs and traces.
+Additionally, the Tempo repository has a [mixin](https://github.com/grafana/tempo/tree/main/operations/tempo-mixin) that includes a
+set of dashboards, rules and alerts.
+Together, these can be used to monitor Tempo in production.
+
+## Instrumentation
+
+Tempo is already instrumented with metrics, logs and traces.
+These can be collected to observe Tempo.
+
+#### Metrics
+
+Tempo is instrumented with [Prometheus metrics](https://prometheus.io/).
+It emits RED metrics for most services and backends.
+The [Tempo mixin](#dashboards) provides several dashboards using these metrics.
+
+#### Logs
+
+Tempo emits logs in the `key=value` ([logfmt](https://brandur.org/logfmt)) format.
+
+#### Traces
+
+Tempo uses the [Jaeger Golang SDK](https://github.com/jaegertracing/jaeger-client-go) for tracing instrumentation.
+As of this writing, only the read path of tempo is instrumented for tracing.
```

Comment:
pieces of the write path are traced. mainly the ingesters? unsure if this is worth mentioning or if it will just complicate things.

Reply:
Might be worth mentioning, but how do you find these traces? The query path posts a helpful request summary, afaik the write path doesn't do this.

Reply:
Hmm, using automatic logging I see some traces related to `tempopb.Pusher/PushBytes` and `CAS` (not sure what this is?)

Reply:
I've updated the wording a bit.

Reply:
yeah, i think we're tracing .1% of our write path starting at the ingesters. so i think tracing is poor enough on the write path it's probably not worth mentioning.

`CAS` may be the ring check and set? Maybe ring interactions are traced? I haven't dug into that.